client/mig-console/action_launcher.go

```diff
@@ -357,29 +357,28 @@ times			show the various timestamps of the action
 				fmt.Println(`Invalid times. Expects settimes <start> <stop.)
 examples:
 settimes 2014-06-30T12:00:00.0Z 2014-06-30T14:00:00.0Z
-settimes now +60m
-`)
```

Comment:
I was seeing an error (not a warning, surprisingly) here due to the "redundant newline" at the end of this format string.